docs/source/gbd2017_models/risk_attributable_causes/chronic_kidney_disease/index.rst

```diff
@@ -250,7 +250,7 @@ State and Transition Data Tables
    * - M
      - excess mortality rate (EMR) of moderate CKD
      - :math:`\frac{\text{CSMR_c589}}{\text{prevalence_sequelae_mod.sub_causes.c589}}`
-     - EMR for moderate CKD is 0, given the assumption that moderate CKD is non-fatal only.
```

Comment:
I thought EMR for M and Sev would be the same, both equal to \frac{\text{CSMR_c589}}{\text{prevalence_sequelae_sev.sub_causes.c589 + prevalence_sequelae_mod.sub_causes.c589}}

Reply:
I read @davidbernotas  email and the note says 'As for the issue concerning EMR and the propensity for people to exit from severe stages more frequently than from mild/moderate stages – Abie recommends, and I agree, that we simply use the CSMR for each cause and apply this to all severity splits symmetrically.' 

I probably didn't understand how to update the EMR caulculations and misunderstood 'applying to severity splits symmetrically'. I will update EMR for Mod and Sev calculations with what your comment says above. If it needs to be revisited, I can update at a later time to something else.